server/src/main/java/org/elasticsearch/synonyms/SynonymsManagementAPIService.java

```diff
@@ -91,6 +101,18 @@ public SynonymsManagementAPIService(Client client) {
         this.client = new OriginSettingClient(client, SYNONYMS_ORIGIN);
     }
 
+    /* The synonym index stores two object types:
+    - Synonym rules:
+        - SYNONYM_RULE_ID_FIELD contains the synonym rule ID
+        - SYNONYMS_FIELD contains the synonyms
+        - SYNONYMS_SET_FIELD contains the synonym set the rule belongs to
+        - OBJECT_TYPE_FIELD contains SYNONYM_RULE_OBJECT_TYPE
+        - The id is calculated using internalSynonymRuleId method
+    - Synonym sets:
+        - SYNONYMS_FIELD contains the synonym set name
```

Comment:
Is there a reason we don't use `SYNONYMS_SET_FIELD` for  the synonym set name? It looks to be its natural place

Reply:
Correct - the comment was wrong, I fixed it 👍